php-frontend/build.gradle.kts

```diff
@@ -0,0 +1,28 @@
+plugins {
+  id("org.sonarsource.php.java-conventions")
+  id("org.sonarsource.php.code-style-convention")
+  id("java-library")
+  id("java-test-fixtures")
+}
+
+description = "SonarSource PHP Analyzer :: Frontend"
+
+dependencies {
+  api(libs.sonar.analyzer.test.commons)
```

Comment:
We should avoid this dependency.
Please try move `PHPCheckTest` and `PHPCheckVerifier` to testFixtures - it can be in separate ticket, please create if needed

Reply:
Nice catch! Will do it later in another ticket: https://sonarsource.atlassian.net/browse/SONARPHP-1558